.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,32 @@
-**By submitting this pull request, I confirm the following (please check boxes, eg [X]) _Failure to fill the template will close your PR_:**
+**By submitting this pull request, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`
 
-***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***
+`{Please ensure that your pull request is for the 'development' branch!}`
 
 - [] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
-- [] I have written tests and verified that they fail without my change.
+- [] I have performed relevant testing to verify that they work with my change.
```

Comment:
The `they` reference can be confusing. Maybe rephrase it to: `I have performed relevant testing and, if applicable, I have written tests to verify the feature works in the future`

Reply:
`[] I have tested my proposed changes, and have included unit tests where possible.`

Make the wording less verbose and just put it plainly.